pytorch_lightning/utilities/cli.py

```diff
@@ -470,7 +473,7 @@ def __init__(
         save_config_multifile: bool = False,
         trainer_class: Union[Type[Trainer], Callable[..., Trainer]] = Trainer,
         trainer_defaults: Optional[Dict[str, Any]] = None,
-        seed_everything_default: Optional[int] = None,
+        seed_everything_default: Optional[Union[bool, int]] = True,
```

Comment:
@mauvilsa would it be useful to also annotate this with `int32`?

Reply:
Yes, here it should also be `int32`.